vcd/resource_vcd_cse_kubernetes_cluster_test.go

```diff
@@ -746,6 +754,10 @@ func TestAccVcdCseKubernetesClusterFailure(t *testing.T) {
 	preTestChecks(t)
 	requireCseConfig(t, testConfig)
 
+	if testConfig.Cse.Version == "4.2.2" || testConfig.Cse.Version == "4.2.3" {
+		t.Skip("test does not work with System Administrator")
```

Comment:
These two error messages do not seem to match the if statement.

Reply:
Improved message